pkg/users/account-details.js

```diff
@@ -98,16 +98,27 @@ function get_expire(name) {
 
 export function AccountDetails({ accounts, groups, current_user, user, shells }) {
     const [expiration, setExpiration] = useState(null);
+    const [path, setPath] = useState("");
```

Comment:
Please make this `null` to make the uninitialized state more distinct. Also please call it `lastlogPath`.